src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import ProfileMenu from '../ProfileMenu';
+import SideMenu from '../SideMenu/SideMenu.component';
+import { MenuIcon, SearchIcon, ProfileIcon } from '../../utils/icons';
+
+import '../../utils/styles/generalStyles.css';
```

Comment:
These styles should be in `Header.styles.css`. Also, you could consider using a layout manager like `display: flex` and play with the different parameters that it offers to avoid setting up each with when it's not really needed